src/vtktools.jl

```diff
@@ -580,7 +581,7 @@ function calc_vtk_points_cells(node_coordinates::AbstractArray{<:Any,4})
   linear_indices = LinearIndices(size_[2:end])
 
   # Use lagrange nodes as VTK points
-  vtk_points = reshape(node_coordinates, (2, n_points))
+  vtk_points = reshape(node_coordinates, (size(node_coordinates, 1), n_points))
```

Comment:
As before, please leave a short comment explaining this seemingly overcomplex way of setting the number of dimensions.